pageserver/src/tenant_mgr.rs

```diff
@@ -508,6 +508,7 @@ pub struct TenantInfo {
     #[serde_as(as = "DisplayFromStr")]
     pub id: ZTenantId,
     pub state: Option<TenantState>,
+    pub current_physical_size: Option<u64>, // physical size is only included in `tenant_status` endpoint
```

Comment:
Please add new field to the swagger spec too (unfortunately this is manual work for now)